bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,228 @@
+import logging
+import random
+import typing
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import Colours, Lovefest, Roles
+from bot.decorators import with_role
+
+log = logging.getLogger(__name__)
+
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+EMOJI_1 = random.choice(HEART_EMOJIS)
+EMOJI_2 = random.choice(HEART_EMOJIS)
+USER_LOVEFEST = []
+JSON_FILE = open(Path('bot', 'resources', 'VALENTINES', 'bemyvalentine_valentines.json'), 'r', encoding="utf8")
```

Comment:
This shouldn't be left dangling open, load the contents into memory (in the cog's `__init__`) & close it.

Reply:
i did close it in the static methods  once i got what i need.

Reply:
It shouldn't be left open at all.

Reply:
how do i load it into memory

Reply:
Store it as an attribute of the cog when you `__init__`

Reply:
something like this :
JSON_FILE = open(Path('bot', 'resources', 'VALENTINES', 'bemyvalentine_valentines.json'), 'r', encoding="utf8")
VALENTINES = load(JSON_FILE)
bot.add_cog(BeMyValentine(bot, VALENTINES))
JSON_FILE.close()
log.debug("Be My Valentine cog loaded")